tools/coverity/misra.config

```diff
@@ -30,6 +30,10 @@
             deviation: "Rule 3.1",
             reason: "Allow nested comments. Documentation blocks contain comments for example code."
         },
+        {
+            deviation: "Rule 8.6",
```

Comment:
could you discuss with Aniruddha/Alfred. They also ran into similar issues with incline suppression.

Reply:
I messaged him about this offline. Marking as resolved.